landing-page/src/WebView.tsx

```
@@ -0,0 +1,606 @@
+            <Typography mt="8px" ml="6px" fontSize="20px" fontWeight="500" color={color.grey.grey11}>{textConst.monthlyUser}</Typography>
+          </Stack>
+        </Stack>
+      </Stack>
+      <Stack pt="32px" width={fixWidth} flexDirection="row">
+        <Stack height="267px" mx="24px" gap="24px" sx={{position: "sticky", top: 164}} display={displayWidthBelow1k ? "none" : "flex"}>
+          <Typography fontSize="18px" fontWeight="500" color={color.grey.grey11}>{textConst.solutions}</Typography>
+          <Stack gap="24px">
+            {(Object.entries(solutionRefList) as [SolutionKey, any][]).map(([solutionKey, ref]) => {
+              const solution = solutions[solutionKey];
+              return (
+                <Stack flexDirection="row" alignItems="center" gap="10px" sx={{cursor: "pointer"}} onClick={()=> scrollTo(ref)}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)